blog.md

```diff
@@ -0,0 +1,166 @@
+# GitOps Observability - Visualizing the journey of a container
+
+In this blog, I’m going to talk about how I brought a simple web application to run on Kubernetes utilizing a GitOps architecture, and how it helped me understand the process. We’ll also talk about how this led our team to understand the need for visualizing the journey of a container — its creation, its utilization in a K8s resource configuration, its approval to be deployed to a cluster, and so on.
+
+## Project Bedrock
+
+The last few years have seen an incredible rise in popularity for Kubernetes, containers and Infrastructure as code. Kubernetes is created to help with automation, deployment and agile methodologies for building software applications.
+
+My team at Microsoft was starting to build a tool called [Bedrock](https://github.com/microsoft/bedrock) which provides a starting point for automating infrastructure management and establishing GitOps pipelines. Being a developer new to Kubernetes and GitOps, I had a huge learning curve — I first began by trying to understand how a simple web application would run on Kubernetes, without using GitOps. This involves creating the Kubernetes resource configuration from ground up, understand YAML configurations, helm charts and all of that. 
```

Comment:
Perhaps instead of 
> My team at Microsoft was starting to build a tool called [Bedrock](https://github.com/microsoft/bedrock) which provides a starting point for automating infrastructure management and establishing GitOps pipelines.

Try:
> My team at Microsoft started collecting our Kubernetes deployment experiences in a project called [Bedrock](https://github.com/microsoft/bedrock). The effort provides a starting point for automating infrastructure management and establishing GitOps pipelines.

This way folks know Bedrock isn't a **tool**